libs/block-editor/src/lib/extensions/blocks/contentlet-block/contentlet-block.component.ts

```diff
@@ -8,12 +8,13 @@ import { AngularNodeViewComponent } from '../../../NodeViewRenderer';
     encapsulation: ViewEncapsulation.None
 })
 export class ContentletBlockComponent extends AngularNodeViewComponent implements OnInit {
-    data: {
-        title: string;
-        inode: string;
-    };
+    public data;
 
     ngOnInit() {
         this.data = this.node.attrs.data;
     }
+
+    getContentState({ live, working, deleted, hasLiveVersion }) {
```

Comment:
doc.

Reply:
we need to extract this to a util. to be reused.